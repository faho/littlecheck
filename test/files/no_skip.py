#!/usr/bin/python
# Any system should have an sh capable of `command -v`
# REQUIRES: command -v sh

print("Oh, but it is!")
# CHECK: This should never be compared
