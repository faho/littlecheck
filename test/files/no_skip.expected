Failure in no_skip.py:

  The CHECK on line 6 wants:
    This should never be compared

  which failed to match line stdout:1:
    Oh, but it is!

  Context:
    Oh, but it is! <= does not match CHECK 'This should never be compared' on line 6

  when running command:
    /usr/bin/python no_skip.py
